docs/04-resources/06-guides/rsk-cli/deploy.md

```diff
@@ -0,0 +1,223 @@
+---
+sidebar_position: 3
+sidebar_label: Deploy Smart Contract via Rootstock-CLI 
+title: Rootstock-CLI Deploy
+description: "The rsk-cli enables you to deploy your smart contract" 
+tags: [Rootstock-CLI, deploy, guides, mainnet, testnet, rsk, rootstock, dApps, smart contracts, solidity]
+---
+
+### **Step 1: Set Up Your Hardhat Project**
+
+1. **Create a New Hardhat Project**:
+
+Open your terminal and create a new project using Hardhat. You can initialize your Hardhat project by running: 
+
+```
+npx hardhat
+```
+
+2. **Project Structure**:  
+   * After creating the project, your folder structure will look something like this:
+
+```
+├── contracts
+│   └── YourSmartContract.sol
+├── scripts
+│   └── deploy.js
+├── test
+├── hardhat.config.js
+├── package.json
+└── node_modules/
+```
+
+### **Step 2: Add and Compile Your Smart Contract**
+
+1. **Create the Smart Contract**:  
+   * Inside the contracts folder, create a new Solidity file (e.g., ContactInfo.sol) and write your smart contract code.
+
+```
```

Comment:
Always add code block format

Reply:
This is not clear. Kindly specify exactly what you would like me to change.